src/encoded/types/file.py

```diff
@@ -136,12 +150,19 @@ def __name__(self):
             return self.uuid
         return properties.get(self.name_key, None) or self.uuid
 
+
     def unique_keys(self, properties):
         keys = super(File, self).unique_keys(properties)
         if properties.get('status') != 'replaced':
             if 'md5sum' in properties:
                 value = 'md5:{md5sum}'.format(**properties)
-                keys.setdefault('alias', []).append(value)
+                if self.registry[CONNECTION].get_by_unique_key('alias', value):
+                    if file_is_md5sum_constrained(properties):
+                        value = 'md5:{md5sum}'.format(**properties)
```

Comment:
Why are you setting this three times?

Reply:
removed unnecessary value='md5'... lines